examples/Sudoku1.py

```diff
@@ -90,19 +85,16 @@
 sudokuout = open('sudokuout.txt','w')
 
 # The solution is written to the sudokuout.txt file 
-for r in Rows:
-    if r == "1" or r == "4" or r == "7":
+for r in ROWS:
+    if r == 1 or r == 4 or r == 7:
```

Comment:
not really that important but... 

```
r in [1, 4, 7]:

```

may look nicer?

Reply:
Agreed!

offtopic: This is my first pull request, so I don't know what is best practice here. Do I have to create a new pull request that incorporates your suggestions?

Reply:
You don't need to create another Pull Request as long as you don't close this one. Just do the necessary commits in the same branch and push your changes. They will appear in this PR.

Reply:
Thx, I submitted the changes.